packages/crashlytics/lib/index.d.ts

```diff
@@ -78,6 +78,51 @@ export namespace FirebaseCrashlyticsTypes {
      *
      */
     isCrashlyticsCollectionEnabled: true;
+    /**
+     * Determines whether there are any unsent crash reports cached on the device. The callback only executes
+     * if automatic data collection is disabled.
+     *
+     * #### Example
+     *
+     * ```js
+     * async checkReports() {
+     * // returns boolean value
+     *  const unsentReports = await firebase.crashlytics().checkForUnsentReports();
+     * }
+     *
+     * checkReports();
+     * ```
+     *
+     */
+    checkForUnsentReports(): Promise<boolean>;
+    /**
+     * Deletes any unsent reports on the device. This method only applies if automatic data collection is
+     * disabled.
+     *
+     * #### Example
+     *
+     * ```js
+     * firebase.crashlytics().deleteUnsentReports();
+     * ```
+     *
+     */
+    deleteUnsentReports(): void;
```

Comment:
```suggestion
    deleteUnsentReports(): Promise<void>;
```

As the native call is async lets make this one return a Promise (JS method also needs updating)